sema4ai/vscode-client/src/deepLink.ts

```diff
@@ -8,8 +8,13 @@ export const DL_APP_BASE_URI_CONTROLLER_VSCODE = `${DL_APP_PROTOCOL_URI_FULL}${D
 
 export const DL_API_ID_IMPORT_ACTION_PACKAGE_FROM_FOLDER = "import-package/folder-path/";
 export const DL_API_ID_IMPORT_ACTION_PACKAGE_FROM_ZIP = "import-package/zip-path/";
+export const DL_API_ID_IMPORT_AGENT_PACKAGE_FROM_ZIP = "import-agent/zip-path/";
```

Comment:
Thinking more about this...
It might be a good moment to change this API a bit just to make it clearer.
Even if we're breaking the backwards compatibility on this, we might not get a better chance in the future to do so.

My suggestion to this:

```suggestion
export const DL_API_ID_IMPORT_ACTION_PACKAGE_FROM_FOLDER = "action/import-package/folder-path/";
export const DL_API_ID_IMPORT_ACTION_PACKAGE_FROM_ZIP = "action/import-package/zip-path/";
export const DL_API_ID_IMPORT_AGENT_PACKAGE_FROM_ZIP = "agent/import-package/zip-path/";
```

We can then sync the releases between the VSCode ext + Studio.
@fabioz what do you think?

Reply:
Makes sense to future-proof this a bit as who knows how many things we end up getting here so we could simplify this even more to something like:

```
export const DL_API_ID_IMPORT_ACTION_PACKAGE_FROM_FOLDER = "action/import/folder/";
export const DL_API_ID_IMPORT_ACTION_PACKAGE_FROM_ZIP = "action/import/zip/";
export const DL_API_ID_IMPORT_AGENT_PACKAGE_FROM_ZIP = "agent/import/zip/"
```
..so essentially <target><action><type>
But in general if you see a need to break comp. good to do it now.

Reply:
I agree. Should be a matter of updating the constants when Studio has this done (so, I guess we can merge and then when Studio has a new release with the breakage we can update in our side).